opacclient/libopac/src/main/java/de/geeksfactory/opacclient/apis/Adis.java

```diff
@@ -263,7 +264,7 @@ public void start() throws IOException {
             Pattern padSid = Pattern
                     .compile(".*;jsessionid=([0-9A-Fa-f]+)[^0-9A-Fa-f].*");
             for (Element navitem : doc
-                    .select("#unav li a, #hnav li a, .tree_ul li a, .search-adv-repeat")) {
+                    .select("#unav li a, #hnav li a, .tree_ul li a, .search-adv")) {
```

Comment:
Have you confirmed that -repeat is not needed in Düsseldorf as well?

Reply:
.search-adv-repeat was added in 51d8a0c, which was specifically for Stuttgart - so I don't think it's needed in Düsseldorf. It seems that after that commit, Stuttgart removed the -repeat part.

Reply:
Right, I was not clever enough to read my own code comment properly 🙈